packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -147,7 +147,12 @@ export abstract class AbstractPureGraphManager {
   // ------------------------------------------- Grammar -------------------------------------------
 
   abstract graphToPureCode(graph: PureModel): Promise<string>;
-  abstract pureCodeToEntities(code: string): Promise<Entity[]>;
+  abstract pureCodeToEntities(
```

Comment:
I disagree I think this method can take that option. SectionIndex was added for specifically grammar cases. Adding it as an option makes sense.

I did think about filtering this out after. However, if we filter by entities we don't know anything about its content as entities are just JSON and thus we could only filter by classiferPath. We don't rely on classiferPath for anything else in studio. We write but never read or rely on them for anything. Don't think we should move to start using them now. The `_type` of the element content is the most reliable way which we can't really use since PMCD is specific to V!. 

thoughts?

Reply:
Oh for some reason I thought it's metamodel we got in the end :). Let's keep it your way then